src/theme/shared/breakpoints.js

```diff
@@ -0,0 +1,14 @@
+export const sizes = {
+  mobile: 0,
```

Comment:
Based on your Slack comment, I'm happy to have this file sit alongside the existing `breakpoint.js` file while we update the integration, but _would_ prefer a clearer name to differentiate; maybe `allBreakpoints` or something?

We should 100% put a ticket in for each repo to update all references, so we can remove the original breakpoint file as soon as possible... although I guess it's really just CRcom, and your new Contact Us repo that would be using it?